content/pt-br/docs/concepts/services-networking/ingress.md

```diff
@@ -0,0 +1,528 @@
+    namespace: external-configuration
+    name: external-config
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Anotação obsoleta
+
+Antes que o recurso IngressClass e o campo `ingressClassName` fossem adicionados no Kubernetes 1.18, as classes Ingress foram especificadas com uma anotação `kubernetes.io/ingress.class` no Ingress. 
+Esta anotação nunca foi formalmente definida, mas foi amplamente apoiada pelos controladores Ingress.
+
+O campo `ingressClassName` mais recente no Ingress é um substituto para essa anotação, mas não é um equivalente direto. 
+Embora a anotação tenha sido geralmente usada para fazer referência ao nome do controlador Ingress que deve implementar o Ingress, o campo é uma referência a um recurso IngressClass que contém a configuração Ingress adicional, incluindo o nome do controlador Ingress.
+
+### Classe Ingress Padrão {#default-ingress-class}
+
+Você pode marcar uma classe Ingress específica como padrão para o seu cluster. 
+Definir a anotação `ingressclass.kubernetes.io/is-default-class` como `true` em um recurso IngressClass garantirá que novos Ingress sem um campo ingressClassName especificado sejam atribuídos a esta `ingressClassName` padrão.
+
+{{< caution >}}
+Se você tiver mais de uma classe Ingress marcada como padrão para o seu cluster, o controlador de admissão impede a criação de novos objetos Ingress que não tenham um `ingressClassName` especificado. 
+Você pode resolver isso garantindo que no máximo uma classe Ingress seja marcada como padrão no seu cluster.
+{{< /caution >}}
+Existem alguns controladores Ingress que funcionam sem a definição de uma `IngressClass` padrão. 
+Por exemplo, o controlador Ingress-NGINX pode ser configurado com uma [flag](https://kubernetes.github.io/ingress-nginx/#what-is-the-flag-watch-ingress-without-class) `--watch-ingress-without-class`. 
+No entanto, é [recomendável](https://kubernetes.github.io/ingress-nginx/#i-have-only-one-instance-of-the-ingresss-nginx-controller-in-my-cluster-what-should-i-do) especificar a `IngressClass` padrão:
+
+{{< codenew file="service/networking/default-ingressclass.yaml" >}}
+
+## Tipos de Ingress
+
+### Ingress backed por um único serviço {#single-service-ingress}
+
+No Kubernetes existem conceitos que permitem expor um único serviço (veja [alternativas](#alternatives)). 
+Você também pode fazer isso com um Ingress especificando um *backend padrão* sem regras.
+
+{{< codenew file="service/networking/test-ingress.yaml" >}}
+
+Se você criá-lo usando `kubectl apply -f`, você deve ser capaz de visualizar o estado do Ingress que você adicionou:
+
+```bash
+kubectl get ingress test-ingress
+```
+
+```
+NAME           CLASS         HOSTS   ADDRESS         PORTS   AGE
+test-ingress   external-lb   *       203.0.113.123   80      59s
+```
+
+Onde `203.0.113.123` é o IP alocado pelo controlador Ingress para satisfazer o Ingress.
+
+{{< note >}}
+Controladores Ingress e balanceadores de carga podem levar um ou dois minutos para alocar um endereço IP. 
```

Comment:
IIRC this address can or cannot be added, and does not control the behavior of Ingress specifically, but things that may consume it, like external-dns. 

I think it is worth to note that not allocating an address doesn't mean your ingress is not working, and it should be tested (my personal opinion here!)